tests/features/header.feature

```diff
@@ -11,3 +11,8 @@ Feature: Page Header
     Then I see the `header`
     And I see the `header` `logo`
     And I see the `header` `search box`
+
+    When I open a `record page`
+    Then I see the `header`
+    And I see the `header` `logo`
+    And I see the `header` `search box`
```

Comment:
Let's rewrite these to use the "`x` in the `y`" syntax used in `record-page.feature`.